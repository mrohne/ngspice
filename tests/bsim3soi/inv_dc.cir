B4SOI  Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
* Modified by Darsen Lu 03/11/2009

.include nmos4p3.mod
.include pmos4p3.mod
.option TEMP=27C noacct

Vpower VD 0 1.5
Vgnd VS 0 0
Vgate Gate 0 0.0
MN0 VS Gate Out VS N1 W=10u L=0.18u
MP0 VD Gate Out VS P1 W=20u L=0.18u

.dc Vgate 0 1.5 0.05
.print dc v(out)
.END
