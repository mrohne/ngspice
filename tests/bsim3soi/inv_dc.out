Note: can't find init file.

Circuit: B4SOI Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)

Warning: Pd = 0 is less than W.
Warning: Ps = 0 is less than W.
Circuit: B4SOI Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
Date: Thu Mar 19 20:54:31  2009


B4SOI Drain Gate Source Back-gate(substrate) Body  Tx  W  L (body ommitted for FB)
DC transfer characteristic  Thu Mar 19 20:54:31  2009
--------------------------------------------------------------------------------
Index   sweep           v(out)          
--------------------------------------------------------------------------------
0	0.000000e+00	1.499923e+00	
1	5.000000e-02	1.499949e+00	
2	1.000000e-01	1.499951e+00	
3	1.500000e-01	1.499845e+00	
4	2.000000e-01	1.499094e+00	
5	2.500000e-01	1.495893e+00	
6	3.000000e-01	1.488441e+00	
7	3.500000e-01	1.476664e+00	
8	4.000000e-01	1.460775e+00	
9	4.500000e-01	1.440490e+00	
10	5.000000e-01	1.415035e+00	
11	5.500000e-01	1.382646e+00	
12	6.000000e-01	1.339924e+00	
13	6.500000e-01	1.279076e+00	
14	7.000000e-01	1.176846e+00	
15	7.500000e-01	8.868706e-01	
16	8.000000e-01	2.308297e-01	
17	8.500000e-01	1.295976e-01	
18	9.000000e-01	8.482178e-02	
19	9.500000e-01	5.850514e-02	
20	1.000000e+00	4.072139e-02	
21	1.050000e+00	2.779857e-02	
22	1.100000e+00	1.820516e-02	
23	1.150000e+00	1.121970e-02	
24	1.200000e+00	6.359835e-03	
25	1.250000e+00	3.253111e-03	
26	1.300000e+00	1.486176e-03	
27	1.350000e+00	6.086077e-04	
28	1.400000e+00	2.277718e-04	
29	1.450000e+00	8.592357e-05	
30	1.500000e+00	4.781260e-05	

CPU time since last call: 0.080 seconds.

Total CPU time: 0.080 seconds.

Current dynamic memory usage = 245760,
Dynamic memory limit = -18361312.

